wire-ios-request-strategy/Sources/Entity Actions/ActionHandler.swift

```diff
@@ -58,7 +58,13 @@ class ActionHandler<T: EntityAction>: NSObject, EntityActionHandler, ZMRequestGe
         let request = self.request(for: action, apiVersion: apiVersion)
 
         request?.add(ZMCompletionHandler(on: context, block: { [weak self] (response) in
-            self?.handleResponse(response, action: action)
+            if response.httpStatus.isOne(of: TooManyRequestsStatusCode, EnhanceYourCalmStatusCode) {
+                // We're being rate limited, put the action back so we can try again
+                // next time the operation loop polls.
```

Comment:
🤔 pulls or polls but again I am not native speaker
```suggestion
                // next time the operation loop pulls.
```

Reply:
I think polling is the correct term

Reply:
Polls in the correct term